pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+packages = ["streetscapes"]
+
+[project]
+name = "streetscapes"
+version = "2024"
+dependencies = [
+    "black",
+    "cloup",
+    "dask",
+    "ephem",
+    "fiona",
+    "geodatasets",
+    "geopandas",
+    "h3",
+    "ipykernel",
+    "mapillary @ git+https://github.com/Urban-M4/mapillary-python-sdk.git",
+    "matplotlib",
+    "numpy",
+    "osmnx",
+    "pandas",
+    "pyarrow",
+    "pyproj",
+    "python-decouple",
+    "rasterio",
+    "seaborn",
+    "timezonefinder",
+    "tqdm",
+]
+authors = []
```

Comment:
Perhaps we can already add ourselves here.

Reply:
Done!